src/main/java/be/seeseemelk/mockbukkit/entity/MushroomCowMock.java

```diff
@@ -79,4 +91,47 @@ public void setStewEffect(@Nullable PotionEffectType type)
 		return EntityType.MUSHROOM_COW;
 	}
 
+	@Override
+	public void shear(Sound.@NotNull Source source)
+	{
+		WorldMock world = (WorldMock) this.getWorld();
+		Location location = this.getLocation();
+		world.playSound(this, org.bukkit.Sound.ENTITY_MOOSHROOM_SHEAR, AdventureConverters.soundSourceToCategory(source), 1.0F, 1.0F);
+		Cow cow = world.spawn(this.getLocation(), Cow.class, null, CreatureSpawnEvent.SpawnReason.CUSTOM, true, false);
+//		world.spawnParticle(Particle.EXPLOSION_NORMAL, location.getX(), location.getY() + 0.5, location.getZ(), 1);
```

Comment:
Why is this in comments

Reply:
Not implemented, added a note in the code.